deploy/README.md

```diff
@@ -1,31 +1,180 @@
 # Deploying the Arrebol Service
 
-The Arrebol service has two main components, the Arrebol Server and the Arrebol Worker. This document describes how to install the Arrebol server component. Before installing the Arrebol Server, please install the [Worker](../worker/deploy).
+This document provides an easy way to deploy a Arrebol Service. The Arrebol has two main components, the **Arrebol Server** and the **Worker Node**. Then, before installing the Arrebol Server, please deploy a [Worker Node](../worker/deploy).
 
-## Requirements
+## Overview
 
-The Docker service can be deployed as a docker container based on the from the [Arrebol docker image](https://hub.docker.com/repository/docker/ufcglsd/arrebol). To be able to run the Arrebol container, install the Docker Engine. To install it, in an apt-based environmnet, checkout this repository and run the `setup.sh` script as follow:
+The Arrebol Service manager the job execution in the worker nodes and monitor the job state. It mean that Arrebol uses the Worker Node resources to run the jobs.
```

Comment:
manage*.... In short, the Arrebol uses the Worker Node resources to run the jobs (I guess this way sounds better)

Reply:
manages*

Reply:
Done.